src/System.Diagnostics.FileVersionInfo/tests/NativeConsoleApp/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I would block these from building at all from our normal build given we are just using the checked in binaries and they aren't useful to build every time.